lib/camera.ts

```diff
@@ -102,11 +109,6 @@ class Camera {
     utils.cleanup(() => {
       this.stopRtsp();
       var stop = new Date().getTime() + 2000;
-      while (new Date().getTime() < stop) {
```

Comment:
I believe this is only necessary because of the driver unload below, which was commented out. I think it's a good idea not to get involved with module loading/unloading and leave this to the user if necessary.